### PR TITLE
chore(docs): add mako to readme

### PR DESCRIPTION
**Pull request by heygsc**
I just found that Mako is using resolver...

### CODE REVIEW

**Comment by Boshen** [comment]
They are stuck on an older version :-/ https://github.com/umijs/mako/blob/c08821ddbc0a80588ec104e0c382a0bf8fd7df3b/crates/mako/Cargo.toml#L25-L26

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ See more at [oxc.rs](https://oxc.rs)!
 
 - [Rolldown] uses the [oxc][docs-oxc-url] crate for parsing and transformation.
 - [Nova engine](https://trynova.dev) uses the [oxc][docs-oxc-url] crate for parsing.
-- [Rolldown][rolldown], [Biome][biome] and [swc-node](https://github.com/swc-project/swc-node) uses the [oxc_resolver][docs-resolver-url] crate for module resolution.
+- [Rolldown][rolldown], [Mako][mako], [Biome][biome] and [swc-node](https://github.com/swc-project/swc-node) uses the [oxc_resolver][docs-resolver-url] crate for module resolution.
```
Comment (Boshen) [inline README.md:31]:
```suggestion
- [Rolldown][rolldown], Biome][biome], [swc-node](https://github.com/swc-project/swc-node) and [Mako][mako] uses the [oxc_resolver][docs-resolver-url] crate for module resolution.
```

**Comment by heygsc** [comment]
`[Biome]`missed a left parenthesis🤣

**Comment by heygsc** [comment]
> `[Biome]`missed a left parenthesis🤣

I added a commit in my branch, but it's not displayed here in PR... I don't know why...

File: README.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ See more at [oxc.rs](https://oxc.rs)!
 
 - [Rolldown] uses the [oxc][docs-oxc-url] crate for parsing and transformation.
 - [Nova engine](https://trynova.dev) uses the [oxc][docs-oxc-url] crate for parsing.
-- [Rolldown][rolldown], [Biome][biome] and [swc-node](https://github.com/swc-project/swc-node) uses the [oxc_resolver][docs-resolver-url] crate for module resolution.
+- [Rolldown][rolldown], Biome][biome], [swc-node](https://github.com/swc-project/swc-node) and [Mako][mako] uses the [oxc_resolver][docs-resolver-url] crate for module resolution.
```
Comment (Boshen) [inline README.md:31]:
```suggestion
- [Rolldown][rolldown], [Biome][biome], [swc-node](https://github.com/swc-project/swc-node) and [Mako][mako] uses the [oxc_resolver][docs-resolver-url] crate for module resolution.
```

**Pull request merged**